pcsx2/GS/Renderers/Metal/GSDeviceMTL.mm

```diff
@@ -89,7 +89,7 @@ static constexpr bool IsCommandBufferCompleted(MTLCommandBufferStatus status)
 	, m_dev(nil)
 {
 	m_backref->second = this;
-	m_mipmap = theApp.GetConfigI("mipmap");
+	m_mipmap = GSConfig.HWMipmap;
```

Comment:
Seems that compile fails here
https://github.com/PCSX2/pcsx2/actions/runs/3725357872/jobs/6318116478#step:11:855
maybe needs a cast?